src/Language/PureScript/Scheme/CodeGen/Transpiler.hs

```diff
@@ -0,0 +1,31 @@
+module Language.PureScript.Scheme.CodeGen.Transpiler where
+
+import Language.PureScript.CoreFn.Module      (Module(..))
+import Language.PureScript.CoreFn.Ann         (Ann)
+import Language.PureScript.CoreFn.Expr        (Expr(..), Bind(..))
+import Language.PureScript.Names              (runIdent)
+import Language.PureScript.AST.Literals       (Literal(..))
+import Language.PureScript.Scheme.CodeGen.AST (AST(..))
+
+
+moduleToScheme :: Module Ann -> [AST]
+moduleToScheme (Module sourceSpan comments name path
```

Comment:
You can avoid complaints from the compiler by prefixing underscores to the names:
```suggestion
moduleToScheme (Module _sourceSpan _comments _name _path
```